packages/live-share-media/src/MediaSessionExtensions.ts

```diff
@@ -116,6 +120,15 @@ export interface ExtendedMediaMetadata extends MediaMetadata {
     liveStream?: boolean;
 }
 
+/**
+ * The reasons why an action is ignored.
+ *
+ * @remarks
+ * Is `localusersuspended` when the action updated the group state, but ignored locally due to the local client being suspended.
+
+ */
+export type ExtendedMediaSessionActionIgnore = "localusersuspended";
```

Comment:
Any reason why this is a type rather than enum?

Reply:
Following existing pattern for action details extensions